cmd/login/cmd.go

```diff
@@ -236,6 +236,13 @@ func run(cmd *cobra.Command, argv []string) {
 		(cfg.FedRAMP && token == "") ||
 		fedramp.IsGovRegion(arguments.GetRegion()) ||
 		config.IsEncryptedToken(token) {
+		// Display error to user if they attempt to log into govcloud without a region specified (fixes OCM-5718)
```

Comment:
@hunterkepley
what if users supply "cfg.FedRAMP" or config.IsEncryptedToken(token) == true?
Since only of the conditions need to truthy to be considered FedRamp, would they all require the fedramp region argument to be passed?